### DISCUSSION TITLE
How to manually pass a context via div.getContext("2d") to PIXI.js?

### DISCUSSION THREAD

**Discussion by ProgrammingLife** [pixijs/pixijs#8079] [Q&A]
I wanna use Pixi for server (non-gpu, node.js) rendering. As for render engine I pick ```skia-canvas```, it can draw on passed context via ```getContext("2d")```. But how I can pass that variable to PIXI to avoid browser-checking, etc?

**Comment by themoonrat** [comment]
If you are creating via Application, or creating your own Renderer, then the `view` settings property can be used to pass in a shared canvas; is this sufficient?
